src/types.ts

```diff
@@ -41,8 +41,17 @@ export type RunConfig = {
   cypress: CypressConfig;
 } & NpmConfigContainer &
   PathContainer &
-  ResultPathContainer;
+  ResultPathContainer &
+  ArtifactsContainer;
 
 export type ResultPathContainer = {
   resultsDir: string;
 };
+
+export type ArtifactsContainer = {
+  artifacts: {
+    remain: {
```

Comment:
According to the ticket. But open to any other name suggestions:
```suggestion
    retain: {
```

Reply:
Ah... Cleaning glasses...